src/dataServices/apiDataService.ts

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable class-methods-use-this */
+
+import {
+    ConditionalDeleteResourceRequest,
+    CreateResourceRequest,
+    DeleteResourceRequest,
+    GenericResponse,
+    PatchResourceRequest,
+    Persistence,
+    ReadResourceRequest,
+    ResourceNotFoundError,
+    UpdateResourceRequest,
+    vReadResourceRequest,
+} from 'fhir-works-on-aws-interface';
+import axios from 'axios';
+
+export class ApiDataService implements Persistence {
+    updateCreateSupported: boolean = false;
+
+    // TODO: Pull this value from AWS Param Store
+    private INTEGRATION_TRANSFORM_URL: string = '';
```

Comment:
empty string is not a valid init value
```suggestion
    private INTEGRATION_TRANSFORM_URL: string;
```

Reply:
updated